src/pages/saksbehandling/revurdering/bunnknapper/RevurderingBunnknapper.tsx

```diff
@@ -0,0 +1,45 @@
+import { Hovedknapp, Knapp } from 'nav-frontend-knapper';
+import React from 'react';
+
+import { useI18n } from '~lib/hooks';
+
+import sharedI18n from '../../../saksbehandling/steg/sharedI18n-nb';
+
+import styles from './revurderingBunnknapper.module.less';
+
+export const RevurderingBunnknapper = (props: {
+    onTilbakeClick(): void;
+    onNesteClick?(): void;
```

Comment:
Vi kan kanskje gjøre nesteknappgreiene litt tydeligere for konsumentn hvis vi endrer til 

```suggestion
    onNesteClick: "submit" | () => void;
```

og så endrer knappkoden til

```ts
                <Hovedknapp
                    spinner={props.onNesteClickSpinner}
                    onClick={props.onNesteClick === "submit" ? undefined : props.onNesteClick}
                    htmlType={props.onNesteClick === "submit" ? 'submit' : 'button'}
                >
```